src/include/common/rpp/handle.hpp

```diff
@@ -66,12 +66,13 @@ using rocblas_handle_ptr = RPP_MANAGE_PTR(rocblas_handle, rocblas_destroy_handle
 struct Handle : rppHandle
 {
     Handle();
-    Handle(size_t nBatchSize);
+    Handle(size_t nBatchSize, Rpp32u numThreads);
```

Comment:
instead of have 2 different constructors, can't you define one with default value for numThreads?

Reply:
We're looking into making this change too. Setting the default value of numThreads = batchSize if no value is passed.

Reply:
Done
Here numThreads is a unsigned int data type, so the min value possible is 0
Already added a case when numThreads == 0, it will be defaulted to batchsize